### ISSUE TITLE
mp4dash.py  : No content protection setup for clearkey?

### ISSUE THREAD

**Issue by sanju-c** [axiomatic-systems/Bento4#501]
https://github.com/axiomatic-systems/Bento4/blob/master/Source/Python/utils/mp4-dash.py 
I believe that clearkey option is not checked for adding content protection. Line 450 and line 506 :
```
 # setup content protection
            if options.encryption_key or options.marlin or options.playready or options.widevine:
                AddContentProtection(options, adaptation_set, video_tracks, all_audio_tracks + all_video_tracks)
```
isn't the check for options.clearkey missing here?
ContentProtection element is not being added to the mpd when I run mp4dash with clearkey option. Please let me know If I am missing out something here!

**Comment by barbibulle** [comment]
You're right. But in practice, it is unlikely you'll be missing the content protection entry, because if `options.encryption_key` is set (meaning that you're encrypting at the same time as packaging). This would only be an issue if you're using the `--clearkey ` options by itself without encrypting as well (for instance if you used a pre-encrypted input). I'll fix this in the next release, in case that's the scenario you're facing.

**Comment by sanju-c** [comment]
Yes , that's exactly the scenario that I am facing (i.e using pre-encrypted input). Thanks for looking into it.
